### PR TITLE
Fix #181: vagrant-service-manager version 1.0.2 release

### LINKED ISSUE #181: vagrant-service-manager 1.0.2 release
- [x] #178 
- [x] #173 
- [x] #169 
- [x] #176 
- [x] #70 
- [x] #177 
- [x] #179 
- [x] #180 
- [x] https://github.com/projectatomic/vagrant-service-manager/pull/193
- [x] https://github.com/projectatomic/vagrant-service-manager/issues/184
- [x] #155  
- [x] #191  
- [x]  #181  
- [x] #198  
- [x] #187  
- [x] https://github.com/projectatomic/vagrant-service-manager/issues/200

### PR DESCRIPTION
**Pull request by navidshaikh**
Fixes #181

### CODE REVIEW

**Comment by navidshaikh** [comment]
do not merge, it needs update.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Changelog
 
 ## Unreleased
+
+## v1.0.2 Apr 14, 2016
```
Comment (brgnepal) [inline CHANGELOG.md]: Shouldn't it be `May 9`?

File: vagrant-service-manager.spec [hunk 2]
```diff
@@ -88,6 +88,9 @@ popd
 %{vagrant_plugin_instdir}/.gitattributes
 
 %changelog
+* Thu Apr 14 2016 Navid Shaikh - 1.0.2-1
```
Comment (brgnepal) [inline vagrant-service-manager.spec]: Shouldn't it be `Mon May 9`?

**Comment by brgnepal** [comment]
LGTM with fixed typo date errors

**Comment by bexelbie** [comment]
LGTM will date changes per @budhrg

**Comment by navidshaikh** [comment]
Updated the PR as per review and added the changelog in spec file as well.

**Pull request merged**